be/src/pipeline/pipeline_tracing.h

```diff
@@ -52,6 +52,7 @@ struct ScheduleRecord {
 using OneQueryTraces = moodycamel::ConcurrentQueue<ScheduleRecord>;
 
 // belongs to exec_env, for all query, if enable
+// curl http://{host}:{web_server_port}/api/running_pipeline_tasks
```

Comment:
`PipelineTracerContext` has nothing to do with running_pipeline_tasks. It's a experimental function WIP